crates/alerter/src/main.rs

```diff
@@ -214,20 +218,22 @@ impl Alerter {
         ) {
             return Ok(());
         }
-        for order in &self.open_orders {
-            if !self.order_has_minimum_age(order) {
-                continue;
-            }
-
-            if self
+        for i in 0..self.open_orders.len() {
```

Comment:
Does Rust 2021 borrow checker let you do:
```
for (order, time) in self.open_orders.iter_mut() {
    // ...
}
```

And figure out that only `open_orders` is borrowed mutably (so you can do stuff with other fields in `self`)? Might make the code slightly more succinct (less array indexing).

At least I have some vague memory that this changed with the new 2021 edition.

Fine as is though.

Reply:
I had that first but didn't work.